### ISSUE TITLE
[0.2] Fatal error: Call to undefined method Raven_Client::setRelease()

### ISSUE THREAD

**Issue by Jean85** [getsentry/sentry-symfony#6]
I just upgraded to 0.2, and it goes on this error. #5 seems to be responsible, in which @dcramer said that a patch release of sentry was needed... Maybe this bit was missing?

Raw stack trace:

```
Symfony\Component\Debug\Exception\FatalThrowableError: Fatal error: Call to undefined method Raven_Client::setRelease()
  Module "app.php", line 28
    $response = $kernel->handle($request);
  Module "bootstrap.php.cache:Symfony\Component\HttpKernel\Kernel", line 2306, in handle
    return $this->getHttpKernel()->handle($request, $type, $catch);
  Module "bootstrap.php.cache:Symfony\Component\HttpKernel\HttpKernel", line 2961, in handle
    return $this->handleException($e, $request, $type);
  Module "bootstrap.php.cache:Symfony\Component\HttpKernel\HttpKernel", line 3025, in handleException
    $this->dispatcher->dispatch(KernelEvents::EXCEPTION, $event);
  Module "EventDispatcher.php:Symfony\Component\EventDispatcher\EventDispatcher", line 42, in dispatch
    if ($listeners = $this->getListeners($eventName)) {
  Module "ContainerAwareEventDispatcher.php:Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher", line 127, in getListeners
    $this->lazyLoad($eventName);
  Module "ContainerAwareEventDispatcher.php:Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher", line 181, in lazyLoad
    $listener = $this->container->get($serviceId);
  Module "bootstrap.php.cache:Symfony\Component\DependencyInjection\Container", line 2107, in get
    $service = $this->$method();
  Module "appProdProjectContainer.php:appProdProjectContainer", line 3609, in getSentry_ExceptionListenerService
    return $this->services['sentry.exception_listener'] = new \Sentry\SentryBundle\EventListener\ExceptionListener($this->get('sentry.client'));
  Module "bootstrap.php.cache:Symfony\Component\DependencyInjection\Container", line 2107, in get
    $service = $this->$method();
  Module "appProdProjectContainer.php:appProdProjectContainer", line 3591, in getSentry_ClientService
    $instance->setRelease(NULL);
```

**Comment by dcramer** [comment]
This should be part of the reqs unless I got the version wrong. Are you running sentry/sentry 0.17?






On Tue, May 17, 2016 at 8:50 AM -0700, "Alessandro Lai" <notifications@github.com> wrote:












I just upgraded to 0.2, and it goes on this error. #5 seems to be responsible, in which @dcramer said that a patch release of sentry was needed... Maybe this bit was missing?



Raw stack trace:

Symfony\Component\Debug\Exception\FatalThrowableError: Fatal error: Call to undefined method Raven_Client::setRelease()
  Module "app.php", line 28
    $response = $kernel->handle($request);
  Module "bootstrap.php.cache:Symfony\Component\HttpKernel\Kernel", line 2306, in handle
    return $this->getHttpKernel()->handle($request, $type, $catch);
  Module "bootstrap.php.cache:Symfony\Component\HttpKernel\HttpKernel", line 2961, in handle
    return $this->handleException($e, $request, $type);
  Module "bootstrap.php.cache:Symfony\Component\HttpKernel\HttpKernel", line 3025, in handleException
    $this->dispatcher->dispatch(KernelEvents::EXCEPTION, $event);
  Module "EventDispatcher.php:Symfony\Component\EventDispatcher\EventDispatcher", line 42, in dispatch
    if ($listeners = $this->getListeners($eventName)) {
  Module "ContainerAwareEventDispatcher.php:Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher", line 127, in getListeners
    $this->lazyLoad($eventName);
  Module "ContainerAwareEventDispatcher.php:Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher", line 181, in lazyLoad
    $listener = $this->container->get($serviceId);
  Module "bootstrap.php.cache:Symfony\Component\DependencyInjection\Container", line 2107, in get
    $service = $this->$method();
  Module "appProdProjectContainer.php:appProdProjectContainer", line 3609, in getSentry_ExceptionListenerService
    return $this->services['sentry.exception_listener'] = new \Sentry\SentryBundle\EventListener\ExceptionListener($this->get('sentry.client'));
  Module "bootstrap.php.cache:Symfony\Component\DependencyInjection\Container", line 2107, in get
    $service = $this->$method();
  Module "appProdProjectContainer.php:appProdProjectContainer", line 3591, in getSentry_ClientService
    $instance->setRelease(NULL);

**Comment by wslaghekke** [comment]
I'm having the same problem.
The setRelease method is not included in the 0.17.0 client, but i do see it on the master branch of sentry/sentry-php

**Comment by dcramer** [comment]
Sorry about that. Resolved in 0.2.1

**Comment by Jean85** [comment]
Can confirm that 0.2.1 fixed the issue! :+1:
